src/xenia/xbox.h

```diff
@@ -382,6 +381,13 @@ enum class XContentType : uint32_t {
   kCommunityGame = 0x02000000,
 };
 
+enum class XNotificationID : uint32_t {
+  kXn_sys_ui = 0x00000009,
```

Comment:
`Xn_` is not needed since it's a `enum class`, but if it was just an `enum` the prefix would be `kXNotificationID`. Also, `_` is rarely used in Xenia in `kCamelCase` cases (mostly when there are numbers to separate), and also the case is random in this commit. Also, `System` doesn't need to be shortened.

Reply:
I've renamed the enum variables as requested. For example, changed `kXn_sys_ui` to `kSystemUI`.